### PR TITLE
changed <h2> to <legend>

### PR DESCRIPTION
**Pull request by math-GH**
fixes parts of #3673

Changes proposed in this pull request:

- `h2` to `<legend>`

How to test the feature manually:

1. same headline layout as on other pages

Pull request checklist:

- [ ] clear commit messages
- [ ] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/edge/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

**Review by Frenzie — COMMENTED** [review]
Some minor whitespace issues, looks otherwise good to me at first glance.

File: app/views/extension/index.phtml [hunk 1]
```diff
@@ -4,12 +4,10 @@
 	<a href="<?= _url('index', 'index') ?>"><?= _t('gen.action.back_to_rss_feeds') ?></a>
 
 	<h1><?= _t('admin.extensions.title') ?></h1>
-
+    <h2><?= _t('admin.extensions.system') ?></h2>
```
Comment (Alkarex) [inline app/views/extension/index.phtml]: Please mind the white-space indenting and use the same than the rest of the file, i.e. tabs

**Comment by Alkarex** [comment]
@mathContao Thanks. Did you have a chance to test with all the themes? I will give it a try tomorrow

File: app/views/extension/index.phtml (hunk 1 above)
Comment (math-GH) [reply app/views/extension/index.phtml]: sorry. Now it is corrected

**Comment by math-GH** [comment]
while I checked some of the themes I found some pages with bad legend/h2 structure and fixed it.
I think it is now fine

**Comment by Alkarex** [comment]
Thanks 👍🏻

**Pull request merged**